### DISCUSSION TITLE
no sound mp4 video

### DISCUSSION THREAD

**Discussion by zoehir** [jely2002/youtube-dl-gui#296] [Q&A] [answered]
Hi,

First of all i want to thank you for the great app. Yesterday i coudn t download a youtube video in MP4 format. When i want to play it in quicktime or want to edit it in fcpx than there is no sound only video. Please can you help me with this problem?

**Comment by jely2002** [comment, marked as answer]
This was probably related to a mix of codecs that weren't working nicely with the player you were using (quicktime). In the current version of the app this is mostly fix and specific codecs can be selected to make sure it also works in Quicktime, think H264 video and AAC audio.
